### ISSUE TITLE
Header Banner Scrolls Horizontally 

### ISSUE THREAD

**Issue by davidc14** [DestinyItemManager/DIM#4164]
If I'm running DIM in a window that is narrow enough to require horizontal scrolling, it is observed that the header banner scrolls along with other content on the page. Thus leaving a large gap in the UI, and making some buttons inaccessible. 

Repro:
- Load DIM - Browser used was Chrome 76 (current stable) on Windows 10 1903
- Make window narrow enough to require horizontal scrolling
- Scroll to the right
- Observe issue

Expected outcome:
Header would not scroll, and would instead fill the current width of the window 

<img width="629" alt="Annotation 2019-09-01 223228" src="https://user-images.githubusercontent.com/963146/64082604-6ab29900-cd09-11e9-8504-18d654ba02d8.png">

**Comment by davidc14** [comment]
This seems to still be occurring in the latest beta build.

**Comment by bhollis** [comment]
Yeah, I had to revert the fix because we were close to prod release and the fix broke iOS
